fw/http.c

```diff
@@ -4918,62 +4918,40 @@ tfw_h1_resp_adjust_fwd(TfwHttpResp *resp)
 	tfw_http_resp_fwd(resp);
 }
 
-/**
- * Error happen, current request @req will be discarded. Connection should
- * be closed after response for the previous request is sent.
- *
- * Returns true, if the connection will be automatically closed with the
- * last response sent. Returns false, if there are no responses to forward
- * and manual connection close action is required.
- */
-static bool
-tfw_http_req_prev_conn_close(TfwHttpReq *req)
-{
-	TfwCliConn *cli_conn = (TfwCliConn *)req->conn;
-	TfwHttpReq *last_req = NULL;
-	struct list_head *prev;
-
-	spin_lock(&cli_conn->seq_qlock);
-	/*
-	 * The request may be not stored in any lists. There are several reasons
-	 * for this:
-	 * - Error happened during request parsing. Client connection is alive.
-	 * - Error happened during response processing, but the client
-	 * connection is already closed, and the request is marked as dropped.
-	 */
-	prev = (!list_empty(&req->msg.seq_list)) ? req->msg.seq_list.prev
-						 : cli_conn->seq_queue.prev;
-	if (prev != &cli_conn->seq_queue) {
-		last_req = list_entry(prev, TfwHttpReq, msg.seq_list);
-		tfw_http_req_set_conn_close(last_req);
-	}
-
-	spin_unlock(&cli_conn->seq_qlock);
-
-	return last_req;
-}
```

Comment:
I remove this function because it is used only when reply == false. But now if no reply is needed tempesta immediately close connection with TCP RST, so this function is used now!